octobot_evaluators/evaluator/social_evaluator.py

```diff
@@ -46,16 +46,20 @@ def load_config(self):
             self.set_default_config()
 
     # Override if no service feed is required for a social evaluator
-    async def start(self) -> None:
+    async def start(self, bot_id: str) -> bool:
         if self.SERVICE_FEED_CLASS is None:
             self.logger.error("SERVICE_FEED_CLASS is required to use a service feed. Consumer can't start.")
         else:
             try:
                 from octobot_services.api.service_feeds import get_service_feed
-                get_service_feed(self.SERVICE_FEED_CLASS).update_feed_config(self.specific_config)
-                await get_chan(self.SERVICE_FEED_CLASS.FEED_CHANNEL.get_name()).new_consumer(self._feed_callback)
+                service_feed = get_service_feed(self.SERVICE_FEED_CLASS, bot_id)
+                if service_feed is not None:
```

Comment:
Should we warn when the `service_feed` is `None` ?

Reply:
We can add a debug message if you think it's necessary. I don't think it is since the only case it would be None is on backtesting when there is no feed available in backtesting mode witch is 100% the case, it would the create useless warnings

Reply:
Oh ok we shouldn't then :+1:

Reply:
And there already is the "Evaluator not started" debug log